web/src/app/routing.module.ts

```diff
@@ -30,6 +30,7 @@ import {EditSurveyComponent} from './pages/edit-survey/edit-survey.component';
 import {EditJobComponent} from './pages/edit-survey/edit-job/edit-job.component';
 import {SurveyDetailsComponent} from './pages/create-survey/survey-details/survey-details.component';
 import {EditSurveyModule} from './pages/edit-survey/edit-survey.module';
+import {ShareSurveyComponent} from './pages/create-survey/share-survey/share-survey.component';
```

Comment:
Since `ShareSurveyComponent` is used by both Create and Edit flows, can we move it into app/components/ as well?